pkg/registry/thirdpartyresourcedata/codec.go

```diff
@@ -34,6 +34,7 @@ import (
 	"k8s.io/kubernetes/pkg/apis/extensions/v1beta1"
```

Comment:
https://github.com/kubernetes/kubernetes/compare/master...sitepod:third-party-resource-watch-fix - I'll add to the commit